cds/config.py

```diff
@@ -26,23 +26,63 @@
 
 from __future__ import absolute_import, print_function
 
+import os
+
 
 # Identity function for string extraction
 def _(x):
     """Indentity function."""
     return x
 
+# Database
+SQLALCHEMY_DATABASE_URI = os.environ.get(
+    "SQLALCHEMY_DATABASE_URI",
+    "postgresql+psycopg2://localhost/cds")
+SQLALCHEMY_ECHO = False
+
 # Default language and timezone
 BABEL_DEFAULT_LANGUAGE = 'en'
 BABEL_DEFAULT_TIMEZONE = 'Europe/Zurich'
-I18N_LANGUAGES = [
-]
+I18N_LANGUAGES = []
+
+# Distributed task queue
+BROKER_URL = os.environ.get(
+    "APP_BROKER_URL",
```

Comment:
btw this should be the default option anyway. did you find any problem with specifying on the default value here.